addons/project/views/project_portal_project_task_templates.xml

```diff
@@ -208,7 +208,7 @@
                                         <div class="o_portal_contact_details d-flex flex-column gap-2">
                                             <div class="d-flex justify-content-start align-items-center gap-2">
                                                 <img class="o_avatar o_portal_contact_img rounded" t-att-src="image_data_uri(task.partner_id.avatar_512)"/>
-                                                <h6 class="mb-0" t-out="task.partner_id.name"></h6>
+                                                <h6 class="mb-0" t-out="task.partner_id" t-options='{"widget": "contact", "fields": ["name"]}'></h6>
```

Comment:
display_name is a better choice than a widget, in my opinion. Can you verify this?

Reply:
i have made the changes
thanks for your suggestions :slightly_smiling_face: